docs/frontend-overrides.md

```diff
@@ -37,14 +36,70 @@ export interface CourseGlimpseProps {
 
 You also need to respect the assumptions made by other components that use your overridden version, if you are not overriding a root component.
 
 For example returning `null` might break a layout if the original component never returned such a value, etc. You also need to make sure to avoid conflict with the parameters accepted by the original component.
+
+## Override translation
+When you create an application based on richie, you can encountered two cases about translations:
+1. You created or overridden a react component and create new translation keys
+2. You just want to override a translation in a existing richie component
+
+
+
+### Create new translation keys
+
+Once you created your new component with its translation keys, you have to extract them with the following command:
+```
+  formatjs extract './**/*.ts*' --ignore ./node_modules --ignore './**/*.d.ts' --out-file './i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format crowdin
+```
+
+This command extracts all translations defined in your typescript files then generates a `frontend.json` file in `i18n/` directory. This file is like a pot file, this is the base to create your translations in any language that you want.
+
+As `--format` option indicates, this command generate a file compatible with crowdin. If you want to customize this command to fit your needs, read the [formatjs/cli documentation](https://formatjs.io/docs/tooling/cli/).
+
+Once translations keys are extracted and your local translation are ready, you need to compile these translations. In fact, compilation process first aggregates all translation files found from provided paths then merges them with richie translations according their filename and finally generates an output formatted for `react-intl`. Below, here an example of compilation command:
+
+```
+  node-modules/richie-education/i18n/compile-translations.js ./i18n/locales/*.json
+```
+
+This command looks for all translation files in `i18n/locales` directory then merges files found with richie translation files. You can pass several path patterns. You can also use an `--ignore` argument to ignore a particular path.
+
+### Override an existing translation keys
+
+As explain above, compilation process aggregates translations files then **merges them according their filename**. That means if you want override for example english translation, you just have to create a `en-US.json` file and redefine translation key used by Richie.
+
+Richie uses one file per language, currently, there 4 languages supported:
+
+- English: filename is `en-US.json`
+- French: filename is `fr-FR.json`
+- Canadian french: filename is `fr-CA.json`
+- Spanish: filename is `es-ES.json`
+
+
+For example, richie uses the translation key `components.UserLogin.logIn` for the Log in button. If you want to change this label for english language, you just have to create a translation file `en-US.json` which redefines this translation key:
```

Comment:
```suggestion
For example, richie uses the translation key `components.UserLogin.logIn` for the Log in button. If you want to change this label for the english translation, you just have to create a translation file `en-US.json` which redefines this translation key:
```

Reply:
don't you "just" as it may be offending for someone struggling to make it work